examples/leds/leds_main.c

```diff
@@ -59,6 +45,26 @@ static bool g_led_daemon_started;
  * Private Functions
  ****************************************************************************/
 
+/****************************************************************************
+ * Name: sigterm_action
+ ****************************************************************************/
+
+void sigterm_action(int signo, siginfo_t *siginfo, void *arg)
```

Comment:
add static

Reply:
Yes, it will better. I'll change it and commit again.

Reply:
Please merge the two patch into one, thanks.